### DISCUSSION TITLE
articles/the-story-of-github-and-gitlab

### DISCUSSION THREAD

**Discussion by hutusi** [hutusi/hutusi.github.com#125] [Comments]
https://hutusi.com/articles/the-story-of-github-and-gitlab 

 引子

**Comment by guuuuo** [comment]
Jenkins 拼错了

**Comment by windy** [comment]
重新回顾了一遍这两个黑客驱动的公司的发展史, 非常棒.

**Comment by hutusi** [comment]
> @guuuuo
> Jenkins 拼错了

谢谢，已Fix~~

**Comment by hutusi** [comment]
> @windy
> 重新回顾了一遍这两个黑客驱动的公司的发展史, 非常棒.

谢谢支持:)

**Comment by kang8** [comment]
写的很棒！！！下一篇会是什么呢？

**Comment by hutusi** [comment]
> @kang8
> 写的很棒！！！下一篇会是什么呢？

这是个好问题，我得酝酿酝酿，可能考虑先写写读书笔记 :)

**Comment by maqi1520** [comment]
大佬可以转载到公众号“JS 酷”吗？ 会注明来源作者和原文链接

**Comment by hutusi** [comment]
@maqi1520 你要不在我的公众号上留言吧，我给你开权限，公众号是"胡涂说"。

**Comment by jkl14599** [comment]
good

**Comment by hutusi** [comment]
@jkl14599 😀

**Comment by rencoo** [comment]
喜欢你的文风，不枯燥

**Reply by hutusi** [reply]
听你这么说太开心了！我得多写✊
